playground/src/components/state-machines-details/state-machines-details.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import flowRight from 'lodash/flowRight';
+import {
+  withApplicationContext,
+  Context,
+} from '@commercetools-frontend/application-shell-connectors';
+import { connect } from 'react-redux';
+import {
+  LoadingSpinner,
+  Spacings,
+  Text,
+  LinkButton,
+  ListIcon,
+  CheckBoldIcon,
+  Grid,
+  Constraints,
+} from '@commercetools-frontend/ui-kit';
+import { fetchStateMachine } from './actions';
+
+export class StateMachinesDetails extends React.Component {
+  static displayName = 'StateMachinesDetails';
+  static propTypes = {
+    id: PropTypes.string.isRequired,
+    projectKey: PropTypes.string.isRequired,
+    backToListPath: PropTypes.string.isRequired,
+    // withApplicationContext
+    dataLocale: PropTypes.string.isRequired,
+    // connect
+    fetchStateMachine: PropTypes.func.isRequired,
+  };
+  state = {
```

Comment:
FYI for the reviewers: as a "follow up step" and as an exercise , we're going to lift up the state to the redux state.